kimchi/src/circuits/expr.rs

```diff
@@ -2170,39 +2175,64 @@ pub mod constraints {
 
         /// Raise the value to the given power
         fn pow(&self, p: u64) -> Self;
+
+        fn literal(x: F) -> Self;
+
+        /// Cache item
+        fn cache(&self, cache: &mut Cache) -> Self;
```

Comment:
these don't seem to be arithmetic ops

Reply:
Indeed - I think the name should change now that we get a better idea of what it is.  Any suggestions?

Reply:
`ExprOperations`?

Reply:
sgtm!